### PR TITLE
fix(doc): provide code snippet for quality change in hls

### PR DESCRIPTION
**Pull request by Ragzzy-R**

### CODE REVIEW

File: docs/javascript/v2/features/hls.mdx [hunk 1]
```diff
@@ -173,6 +173,79 @@ function HLSView({ hlsUrl }) {
...
+        if (videoRef.current && hlsUrl) {
+            if (Hls.isSupported()) {
+                let hls = new Hls();
+                console.log("Available quality levels in current stream", hls.levels);
+                setCurrentLevel(hls.levels[0]); //set the first level.
+            }
+        }
+    }, [hlsUrl]);
+
+    function setCurrentLevel(currentLevel) {
```
Comment (raviteja83) [inline docs/javascript/v2/features/hls.mdx:231]: format properly. seems off

Comment (Ragzzy-R) [reply docs/javascript/v2/features/hls.mdx:227]: formatted the entire document with eslint now. let me know if something else is needed.

File: docs/javascript/v2/features/hls.mdx [hunk 2]
```diff
@@ -173,6 +173,73 @@ function HLSView({ hlsUrl }) {
...
+}
+
+function setCurrentLevel(currentLevel) {
+    // make sure we are choosing the right level by checking the
+    // height and width.
+    const newLevel = this.hls.levels.findIndex(
+        (level) => level.height === currentLevel.height && level.width === currentLevel.width
+    );
+    // set it.
+    this.hls.currentLevel = newLevel;
```
Comment (raviteja83) [inline docs/javascript/v2/features/hls.mdx:204]: there is not this.hls here

File: docs/javascript/v2/features/hls.mdx [hunk 3]
```diff
@@ -173,6 +173,73 @@ function HLSView({ hlsUrl }) {
...
+<Tabs id="hlsChangeQuality" items={['JavaScript', 'React']} />{' '}
+
+<Tab id='hlsChangeQuality-0'>
+
+```js
+import Hls from 'hls.js';
+
+function changeQuality() {
+    if (Hls.isSupported()) {
+        let hls = new Hls();
```
Comment (raviteja83) [inline docs/javascript/v2/features/hls.mdx:191]: let's initialise outside.

File: docs/javascript/v2/features/hls.mdx (hunk 2 above)
Comment (Ragzzy-R) [reply docs/javascript/v2/features/hls.mdx]: done

File: docs/javascript/v2/features/hls.mdx (hunk 3 above)
Comment (Ragzzy-R) [reply docs/javascript/v2/features/hls.mdx]: done

File: docs/javascript/v2/features/hls.mdx [hunk 4]
```diff
@@ -181,6 +181,77 @@ function HLSView({ hlsUrl }) {
...
+<Tabs id="hlsChangeQuality" items={['JavaScript', 'React']} />{' '}
+
+<Tab id='hlsChangeQuality-0'>
+
+```js
+import Hls from 'hls.js';
+
+let hls = null;
+const video = document.getElementById('video');
+const browserHasNativeHLSSupport = videoRef.current.canPlayType('application/vnd.apple.mpegurl');
```
Comment (raviteja83) [inline docs/javascript/v2/features/hls.mdx:199]: `video.canPlayType.` btw, this doesn't seem to be used anywhere?

Comment (Ragzzy-R) [reply docs/javascript/v2/features/hls.mdx]: done

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**